apps/full-site-editing/package.json

```diff
@@ -17,31 +17,31 @@
 	"homepage": "https://github.com/Automattic/wp-calypso",
 	"scripts": {
 		"full-site-editing": "calypso-build --source='dotcom-fse'",
-		"dev:full-site-editing": "npm run full-site-editing",
-		"build:full-site-editing": "NODE_ENV=production npm run full-site-editing",
+		"dev:full-site-editing": "yarn run full-site-editing",
+		"build:full-site-editing": "NODE_ENV=production yarn run full-site-editing",
 		"posts-list-block": "calypso-build --source='posts-list-block'",
-		"dev:posts-list-block": "npm run posts-list-block",
-		"build:posts-list-block": "NODE_ENV=production npm run posts-list-block",
+		"dev:posts-list-block": "yarn run posts-list-block",
+		"build:posts-list-block": "NODE_ENV=production yarn run posts-list-block",
 		"blog-posts-block-editor": "calypso-build --source='blog-posts-block/editor'",
 		"blog-posts-block-view": "calypso-build --source='blog-posts-block/view'",
 		"blog-posts-block": "npm-run-all --parallel blog-posts-block-*",
```

Comment:
In case you are wondering, `npm-run-all` uses `yarn` when invoked from a yarn script (https://www.npmjs.com/package/npm-run-all#yarn-compatibility)